src/content/docs/developer-tools/kinde-api/about-m2m-scopes.mdx

```diff
@@ -14,22 +14,39 @@ app_context:
     s: apis
 ---
 
-The Kinde management API uses JSON Web Tokens (JWTs) to authenticate requests. The token’s scopes claim indicates which endpoints can be accessed when calling the management API.
+The Kinde management API uses JSON Web Tokens (JWTs) to authenticate requests. The token’s scopes claim indicates which endpoints can be accessed when calling the API.
 
 ## Example scopes
 
 - `read:users` for fetching user details.
 - `write:users` for modifying user details.
 - `read:roles` or `write:roles` for managing roles.
 
-You can select scopes when you authorize a new application or you can add scopes to an existing application. We recommend adding as few scopes as you need, to maintain API security. 
+You can select scopes when you authorize a new application or you can add scopes to an existing application. We recommend adding as few scopes as you need, to maintain API security.
 
-## Change or add scopes to an application accessing the Kinde Management API
+## Request scopes in the body of an access token request
+
+By default token requests for an authorized application will return all the scopes enabled below. You can also optionally ask for a subset of enabled scopes to be returned by including them in the body of the access token request. You might do this to add more security to access requests, or because you want your users to be very specific about scopes in their requests.
```

Comment:
@clairekinde11 for management API it is unlikely their end users would be requesting the token. So probably
> You might do this to generate a token with limited access in order to minimize risk if the token was compromised